docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,500 @@
+│       │           │       │   └── cert.pem
+│       │           │       └── user
+
+```
+
+There are three sections in the folder structure above to consider:
+- **Orderer organization MSP:** `organizations/ordererOrganizations/ordererOrg1.example.com/msp` folder contains the orderer organization MSP that includes the `cacerts` and  `tlscacerts` folders that you need to create and then copy in the root certificates (`ca-cert.pem`) for the organization CA and TLS CA respectively. If you are using an intermediate CA, you also need to include the corresponding `intermediatecerts` and `tlsintermediatecerts` folders.
+- **Orderer local MSP:**`organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/msp` folder, also known as the orderer local MSP, contains the enrollment certificate and private key for the ordering service `osn1` node. This folder is automatically generated when you enroll the orderer identity with the organization CA.
+- **TLS certificates:** `organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls` folder contains the TLS certificate and private key for the orderer `osn1` node as well as the TLS CA root cert `tls-ca-cert.pem`.  
+
+The `config.yaml` file in the orderer organization MSP and orderer local MSP enables Node OU support for the channel, an important feature that allows the channel to recognize certificates that contain the "admin" OU as an admin identity of the channel. Learn more in the [Fabric CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#nodeous) documentation.
+
+## Prerequisites
+
+Because `osnadmin` commands are performed from the perspective of an ordering service node, the ordering service needs to exist before you can create a channel. You can attempt this tutorial with your existing ordering service or deploy a new ordering service.  If you decide to use `osnadmin` command  with an existing ordering service, the system channel must first be removed from each ordering node before you can create any new channels. Choose whether you want to use your existing ordering service or deploy a new one:
+
+- [Use existing ordering service](#use-existing-ordering-service)
+- [Deploy a new ordering service](#deploy-a-new-ordering-service)
+
+### Use existing ordering service
+
+Before you can take advantage of this feature on a deployed ordering service, you need to remove the system channel from each ordering node in the application channel consenter set. A "mixed mode" of orderers on a channel, where some nodes are part of a system channel and others are not, is not supported. The `osadmin` CLI includes both a `channel list` and a `system-channel remove` command to facilitate the process of removing the system channel. If you prefer to deploy a new ordering service instead, skip ahead to [Deploy a new ordering service](#deploy-a-new-ordering-service).
```

Comment:
There is not a separate "system-channel remove" command, it is simply using the "channel remove" command and specifying the --channel-id of the system channel.

Reply:
Thought I got those! There were four other occurrences so I think I got them all now.